From 98ada6c00c8fd93205ce9206f61bdf14b5a95912 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Pawe=C5=82=20Wegner?= <pawel.wegner95@gmail.com>
Date: Sat, 17 Apr 2021 11:04:13 +0200
Subject: [PATCH] zlib: fix build.

---
 zconf.h.in | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/zconf.h.in b/zconf.h.in
index ae2a309..c21cba3 100644
--- a/zconf.h.in
+++ b/zconf.h.in
@@ -131,7 +131,9 @@ typedef void       *voidpf;
 typedef void       *voidp;
 
 #ifdef HAVE_UNISTD_H    /* may be set to #if 1 by configure/cmake/etc */
-#  define Z_HAVE_UNISTD_H
+#  if ~ HAVE_UNISTD_H + 1
+#    define Z_HAVE_UNISTD_H
+#  endif
 #endif
 
 #ifdef NEED_PTRDIFF_T    /* may be set to #if 1 by configure/cmake/etc */
-- 
2.25.1

